can/interfaces/gs_usb.py

```diff
@@ -144,3 +150,13 @@ def _recv_internal(
     def shutdown(self):
         super().shutdown()
         self.gs_usb.stop()
+        if self.index is not None:
+            # Avoid errors on subsequent __init() by repeating the .scan() and .start() that would then fail here
+            devs = GsUsb.scan()
+            gs_usb = devs[self.index]
+            try:
+                gs_usb.set_bitrate(self.bitrate)
```

Comment:
Is this really necessary? Seems weird to call these methods in `shutdown()`. The comment does not really explain much.

Reply:
Yeah... it unfortunately is. Right now every second time python-can gs_usb interface tries to use the device there is a USBError thrown.

```
Traceback (most recent call last):
  File "C:\Users\Ben Gardiner\src\python-can\can\interfaces\gs_usb.py", line 65, in __init__
    self.gs_usb.start()
  File "C:\Users\Public\miniconda3\Lib\site-packages\gs_usb\gs_usb.py", line 50, in start
    self.gs_usb.reset()
  File "C:\Users\Public\miniconda3\Lib\site-packages\usb\core.py", line 961, in reset
    self._ctx.backend.reset_device(self._ctx.handle)
  File "C:\Users\Public\miniconda3\Lib\site-packages\usb\backend\libusb0.py", line 618, in reset_device
    _check(_lib.usb_reset(dev_handle))
  File "C:\Users\Public\miniconda3\Lib\site-packages\usb\backend\libusb0.py", line 447, in _check
    raise USBError(errmsg, ret)
usb.core.USBError: [Errno None] b'libusb0-dll:err [reset] could not reset device, win error: A device attached to the system is not functioning.\r\n\n'
```

I don't fully understand why though so the hamfisted way to fix it is to do the second open (start) that we know will fail in the shutdown method.

Reply:
I've added what I hope is a more descriptive comment there